spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -73,32 +79,32 @@ interface UpgradeTimeout {
 - `timeoutHeight`: Timeout height indicates the height at which the counterparty must no longer proceed with the upgrade handshake. The chains will then preserve their original channel and the upgrade handshake is aborted.
 - `timeoutTimestamp`: Timeout timestamp indicates the time on the counterparty at which the counterparty must no longer proceed with the upgrade handshake. The chains will then preserve their original channel and the upgrade handshake is aborted.
 
-At least one of the timeoutHeight or timeoutTimestamp MUST be non-zero.
+At least one of the `timeoutHeight` or `timeoutTimestamp` MUST be non-zero.
 
 ### Store Paths
 
-#### Restore Channel Path
+#### Backup Channel Path
```

Comment:
I thought that `backup` had a more close meaning to what this path serves for, which is basically to back up the channel end in case it is needed to restore it later on.

Reply:
I prefer keeping it as `restore` actually

Reply:
All right, cool. I will revert this change then.